### DISCUSSION TITLE
Howto delete all uploaded but not used media files before exporting a liascript project zip?

### DISCUSSION THREAD

**Discussion by michaelkowalczyk** [LiaScript/LiaScript#199] [Q&A] [answered]
When I export the project as zip, the package will contain all uploaded media files, even those which are not used anymore. 
How one can purge all not used media files and keep only the relevant project files?
Thanks for tipps.

**Comment by andre-dietrich** [comment, marked as answer]
Hi, at the moment this has to be done manually from the zip folder ... However, after the integration of zip-files into LiaScript, which will allow packing all materials into one file, we will try to update the LiveEditor as well, with a sidebar that will show a file hierarchy of all files and will give you more control over the content and naming of files ....

**Reply by michaelkowalczyk** [reply]
Thanks for the update and outlook Andre.
